src/payment/payment-methods/cybersource/service.js

```diff
@@ -19,36 +20,47 @@ export function configureApiService(newConfig, newApiClient) {
   apiClient = newApiClient;
 }
 
-export async function sdnCheck(firstName, lastName, city, country) {
+export async function sdnCheck(basketId, firstName, lastName, city, country) {
   const { data } = await apiClient.post(
     `${config.ECOMMERCE_BASE_URL}/api/v2/sdn/search/`,
     {
       name: `${firstName} ${lastName}`,
       city,
       country,
     },
-  );
+  ).catch((error) => {
+    logAPIErrorResponse(error, {
+      messagePrefix: 'SDN Check Error',
```

Comment:
As you did below, do we want to make it clear that this was an error attempting to check SDN, and not that the user didn't pass SDN?  This too can be in a follow-up PR.